app/addons/replication/stores.js

```diff
@@ -77,6 +110,23 @@ const ReplicationStore = FauxtonAPI.Store.extend({
     this._activitySort = sort;
   },
 
+  isReplicateInfoLoading () {
+    return this._fetchingReplicateInfo;
+  },
+
+  getReplicateInfo () {
+    return this._replicateInfo.filter(doc => {
+      return _.values(doc).filter(item => {
+        if (!item) {return null;}
```

Comment:
Minor but can we change this to return `false`?  I feel like it'd be more readable when considering `filter` keeps/discards items based on boolean values.